source/features/linkify-code.tsx

```diff
@@ -17,7 +17,6 @@ function initTitle(): void {
 
 function init(): void {
 	const selectors = [
-		'.js-blob-wrapper',
```

Comment:
This is added in https://github.com/sindresorhus/refined-github/pull/2066/files#diff-49baf2e54681813b16cfb0a3b15c468590e5f5ee7c584af83fecd0beb5856238R47 but honestly I don't understand the reason. Is there any `.js-blob-wrapper` that isn't included in `.blob-wrapper`?

Reply:
There might be 2 reasons for that change (I don't remember):

- backwards compatibility, maybe one selector was left in the code but it wasn't needed
- to support one of the many situations mentioned in that post, so if you remove it make sure the feature still works in every one of those